src/components/BackButton.tsx

```diff
@@ -24,7 +23,6 @@ BackButton.defaultProps = {
 
 const styles = StyleSheet.create({
   container: {
-    paddingLeft: variables.contentPadding + 6, // 6px from the left padding
```

Comment:
I see that you moved this style to be an inline style in the CustomHeader component, why do we need this change? is it backwards compatible for other components that use it?

Reply:
Updated in c4fd878f8760e1507784e5888c8e8086bd4eba2b!